js/client-chat-tournament.js

```diff
@@ -355,10 +355,16 @@
 						this.info.isActive = true;
 					}
 
-					if ('format' in this.updates || 'isJoined' in this.updates) {
+					if ('format' in this.updates) {
 						this.$format.text(window.BattleFormats && BattleFormats[this.info.format] ? BattleFormats[this.info.format].name : this.info.format);
 						this.updateTeams();
 					}
+					if ('customFormat' in this.updates) {
+						this.$format.text(window.BattleFormats && BattleFormats[this.info.customFormat] ? BattleFormats[this.info.customFormat].name : this.info.customFormat);
+					}
+					if ('isJoined' in this.updates) {
```

Comment:
<code>'isJoined' in this.updates</code> was in the same if statement as <code>'format' in this.updates</code>, so the format name was set whenever the user joined the tournament.